source/common/quic/envoy_quic_client_session.cc

```diff
@@ -15,11 +15,12 @@
 namespace Envoy {
 namespace Quic {
 
-// Quic connection doesn't use transport socket object to do handshake. This callback
-// implementation is used by the cert verifier to access Network::Connection interfaces.
-class ConnectionWrapper : public Network::TransportSocketCallbacks {
+// Quic connection doesn't use transport socket object to do handshake. This class is only used to
```

Comment:
nit: Sorry, I meant to suggest removing this first sentence completely.

Reply:
done